.github/scripts/sync_labels.py

```diff
@@ -48,12 +48,13 @@ def get_linked_pr_from_issue_number(repo, number):
 
 def get_linked_issues_based_on_pr_body(repo, number):
     pr = repo.get_pull(number)
-    pattern = fr'Fixes:? (?:#|{repo}#|https://github.com/{repo}/issues/)(\d+)'
-    matches = re.findall(pattern, pr.body)
+    pattern = rf'(?:fix(?:|es|ed)|resolve(?:|d|s)|close(?:|s|d))?(?:{repo}|https://github.com/{repo}/issues/)?(\d+)'
```

Comment:
i think i root caused the issue. `repo` is an instance of `github.Repository.Repository`, if we just interpolate it with f-string, we would get `"Repository(full_name="joe/project_name")`. but what we need is but `"joe/project_name"`. so, i'd suggest:
```py
repo_name = repo.full_name
pattern = rf'(?:fix(?:|es|ed)|resolve(?:|d|s)|close(?:|s|d))?(?:{repo_name}|https://github.com/{repo_name}/issues/)?(\d+)'
```

Reply:
@tchaikov Thanks for that :-) 

Fixed